src/components/app/ProfileDeleteButton.js

```diff
@@ -152,10 +152,6 @@ export class ProfileDeletePanel extends PureComponent<PanelProps, PanelState> {
     }
   };
 
-  onCancelDeletion = () => {
-    this.props.onProfileDeleteCanceled();
```

Comment:
This function is used directly in the render function

Reply:
That makes sense.